docs(bot-onboarding): add the new 'echanges-client'

MessaKami opened the pull request:
add the new echanges-client into the folder echanges-client/echanges-16-12-24 in the file mail-recap.md

MessaKami committed:
docs(bot-onboarding): rename the user_name
rename the user_name to user_lastname into the folder bdd on the file dictionnaire-de-donnees.md

MessaKami committed:
docs(bot-onboarding): add the new echanges-client

aReynier reviewed (commented):
- Regrouper les forums et les channels vocaux au sein d'une même category discord: ce n'est pas posts plutot que forum?
- Template de forum: préciser qu'il s'agit de celui des formations?
- On valide bien le fait de regrouper les stocks de channel et posts en une category? (on souhaitais le séparer pour bonnes pratiques)
- Est-ce qu'on rajoute le fait que les noms de formation et promo sont concaténées?
- La notion de post générique et non générique et comment ils sont ajoutés, pris en compte est il bien évoqué?
- Ajoute-t-on la possibilité d'être dans deux promos pour les personnes qui enchainent les formations?  
- La documentation reçue de Benjamin sera exactement la documentation Discord JS, et Benjamin doit nous envoyer précisemment la partie sur comment les channels sont stockés, le précise t'on?
- Au niveau du MCD, scinder la table user en deux pour avoir une table avec les infos user liées à discord et les infos user non liées à Discord, le rajoute t'on?

MessaKami committed:
docs(bot-onboarding): update the mail recap
update the mail recap for the 16-12-2024 in echanges-client/echanges-16-12-24/mail-recap.md

MessaKami committed:
docs(bot-onboarding): update 1 sentence
update the mail recap for the 16-12-2024 in echanges-client/echanges-16-12-24/mail-recap.md

YohanF1245 reviewed (approved):
There's a little stowaway commit . Changes are done in "dictionnaire-de-données.md" but the pull request name does not mention it.
